client/src/renderer/components/Clipboard/index.jsx

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+import "./styles.scss";
+import { Component, h } from "preact";
+import clipboard from "electron-clipboard-extended";
+
+/**
+ * Stores and lists contents of system clipboard
+ * @extends Component
+ */
+
+export default class Clipboard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { clipboardStack: [] };
+    this.handleTextChanged = this.handleTextChanged.bind(this);
+    this.handleImageChanged = this.handleImageChanged.bind(this);
+  }
+
+  componentDidMount() {
+    clipboard.startWatching();
+    clipboard.on("text-changed", this.handleTextChanged);
+    clipboard.on("image-changed", this.handleImageChanged);
+  }
+
+  componentWillUnmount() {
+    clipboard.stopWatching();
+    clipboard.off("text-changed");
+    clipboard.off("image-changed");
+  }
+
+  handleTextChanged() {
+    this.setState(prevState => {
+      return {
+        clipboardStack: [
+          ...prevState.clipboardStack,
+          new ClipboardItem("txt", clipboard.readText()),
+        ],
+      };
+    });
+  }
+
+  handleImageChanged() {
+    this.setState(prevState => {
+      const img = clipboard.readImage();
+      const imgURI = img.toDataURL();
+      return {
+        clipboardStack: [
+          ...prevState.clipboardStack,
+          new ClipboardItem("img", imgURI),
+        ],
+      };
+    });
+  }
+
+  render(props, state) {
+    return (
+      <ul class="clipboard-list">
```

Comment:
Just a minor name change.

```suggestion
      <ul class="clipboard">
```